app/Services/Adselect.php

```diff
@@ -138,7 +136,7 @@ public function addCampaigns(array $campaings)
 
     public function deleteCampaigns(array $campaignIds)
     {
-        return $this->rpcClient->execute('campaign_delete', $campaignIds);
+//        return $this->rpcClient->execute('campaign_delete', $campaignIds);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS125)